src/chapters/modules/npm.rst

```diff
@@ -45,63 +45,47 @@ you can search for a module by name or desired functionality.
 
    .. figure:: ./figures/readline-sync-npm-page.png
 
+.. _npm-cli:
+
 NPM Command Line Interface (CLI)
 ---------------------------------
 
 The NPM command line tool, **CLI**, is installed with Node. The NPM CLI is used
-in a computer's terminal to install modules into a Node project. Before we can
-talk more about the NPM CLI, however, we need to discuss repl.it and NPM.
-
-We have coded many Node projects inside of repl.it. Repl.it is great. It allows
-us to simulate a development environment WITHOUT having to install any software
-on our computers. As such, it automatically handles much of the work for
-installing external modules.
-
-Let's examine how using the CLI is different when using repl.it.
+in a computer's *terminal* to install modules into a Node project. There is no
+need to panic! You will learn how to use the terminal in a later chapter.
 
-CLI With Local Development Environment
----------------------------------------
+For now, recall that we coded many Node projects inside of repl.it, which
+allows us to simulate a development environment WITHOUT having to install any
+software on our computers. As such, repl.it automatically handles much of the
+work of installing external modules.
 
-The following describes the steps for working with NPM outside of repl.it. Do
-not worry about following along. We are simply reviewing these to familiarize
-you with the general process.
-
-#. `Install Node on your computer <https://nodejs.org/en/download/>`__, which
-   also installs the NPM CLI tool.
-#. Use the CLI tool in a terminal to install modules into your local project.
-   The syntax is ``npm install <package_name>``, and running it downloads the
-   module to your computer and adds an entry into a ``package.json`` file.
-   This entry indicates that your project depends on the module called
-   ``package_name``.
-#. More detailed instructions can be found in the
-   `NPM documentation <https://docs.npmjs.com/downloading-and-installing-packages-locally>`__.
+Instead of typing commands into this mysterious "Terminal" thing, repl.it adds
```

Comment:
I think we should remove this sentence because they don't have a reference for running commands in the terminal yet.

Reply:
Removed.  Thanks.